.pre-commit-config.yaml

```diff
@@ -4,7 +4,3 @@ repos:
     hooks:
     - id: black
       language_version: python3.7
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v1.2.3
-    hooks:
-    - id: flake8
```

Comment:
👎 :trollface:

Reply:
Black forces a lot of the things that flake8 complains about, and if there's an illegal line then black should explode anyway. I can re-add it, it just seemed extraneous when we're already forcing a code style.

Reply:
Yep, I just like flake8. It's a design choice, but I'm on board with it. :smile: